src/components/data-table/config/index.js

```diff
@@ -0,0 +1,74 @@
+export const args = {
+  hover: false,
+  matrix: false,
+  lined: false,
+  zebra: false,
+  collapsedM: false,
+  collapsedS: false,
+  collapsedXS: false,
+};
+
+export const argTypes = {
+  hover: {
+    name: "data-vl-hover",
+    type: { summary: "boolean" },
+    description:
+      "Attribuut wordt gebruikt om een rij te highlighten waneer de gebruiker erover hovert met muiscursor.",
```

Comment:
Uw intro text staat in het engels, deze docs in het nederlands. Ik zou het zo universeel mogelijk maken in het engels.

Reply:
Fixed.